guides/source/action_view_overview.md

```diff
@@ -42,12 +44,11 @@ $ bin/rails generate scaffold article
       [...]
 ```
 
-There is a naming convention for views in Rails. Typically, the views share their name with the associated controller action, as you can see above.
-For example, the index controller action of the `articles_controller.rb` will use the `index.html.erb` view file in the `app/views/articles` directory.
-The complete HTML returned to the client is composed of a combination of this ERB file, a layout template that wraps it, and all the partials that the view may reference. Within this guide, you will find more detailed documentation about each of these three components.
+The naming convention is that the views share their name with the associated controller action. For example the `index.html.erb`, `edit.html.erb`, etc. above.
```

Comment:
Instead of 'The naming convention is that the views...'
Suggestion: 'Following Rails convention, views share their name with the associated controller action.'